xsl/xccdf-resources/openscap.js

```diff
@@ -179,44 +180,27 @@ function sortGroups(groups, key)
     switch(key) {
     case KeysEnum.SEVERITY:
         return ["high", "medium", "low"];
-    case KeysEnum.DISA:
-        return groups.sort(function(a, b){
-            return parseInt(a) - parseInt(b);
-        });
-    case KeysEnum.NIST:
+    case KeysEnum.RESULT:
+	return groups.sort();
+    default:
         return groups.sort(function(a, b){
-            var regex = /(\w\w)-(\d+)(.*)/;
-            var a_parts = regex.exec(a);
-            var b_parts = regex.exec(b);
-            if (a_parts == null)
-                return 1;
-            if (b_parts == null)
-                return -1;
-            var result = a_parts[1].localeCompare(b_parts[1]);
-            if (result != 0) {
-                return result;
-            } else {
-                result = a_parts[2] - b_parts[2];
-                if (result != 0) {
-                    return result;
+            var a_parts = a.split(/[.()-]/);
```

Comment:
I think it doesn't cause real-world performance issue, but If I understand it well
you need to compile these regexps every time you need to compare some two elements.
I don't know if JS can optimize this. I personally prefer move initialization of these regexps out of function what is call many times.